ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -138,11 +139,34 @@ def platform_dependent_defines():
     if os.name == "posix":
         ret.append("-DOS_POSIX=1")
 
-    ret.append({
-        "Linux": "-DOS_LINUX=1",
-        "Darwin": "-DOS_MACOSX=1",
-        # TODO: Windows?
-    }[platform.system()])
+    system = platform.system()
+    if system == "Linux":
+        ret.append("-DOS_LINUX=1")
+    elif system == "Darwin":
+        ret.append("-DOS_MACOSX=1")
+    elif system == "Windows":
+        ret.append("-DOS_WIN=1")
+        ret.append("-DWIN32=1")
+        ret.append("-use-msvc-mangling")
+        msvc_platform = os.environ["PLATFORM"]
```

Comment:
Huh, fair enough, can you add to the exception message something similar to "Only windows MSVC builds are supported"?